client/js/components/tags/TagsList.vue

```diff
@@ -277,23 +277,30 @@ export default {
       this.saveTagTopic(oldParent.id)
     },
 
-    save ({ id, attributes, type }) {
-      if (id === '') {
+    save ({ id, attributes, type, isTitleChanged }) {
+      if (!id || !isTitleChanged) {
+        this.closeEditForm()
+
         return
       }
 
-      console.log('save', { id, attributes, type })
+      const updateMethod = `update${type}`
+      const saveMethod = `save${type}`
 
-      this[`update${type}`]({
-        attributes,
-        id,
-        relationships: this[type][id].relationships,
-        type
-      })
-      this[`save${type}`](id)
-        .then(() => {
-          this.isInEditState = ''
+      if (typeof this[updateMethod] === 'function' && typeof this[saveMethod] === 'function') {
```

Comment:
I am not sure if this is the best approach. Why can it be not a function? 
Because, if its no function, we still can't save. That don't fixes the issue imho.

Reply:
I understand you , but I thought it would be a nice check, especially when the type we receive from the outside could change and may no longer be a function in the FE. However, I understand your perspective and c[an remove it if you want](https://github.com/demos-europe/demosplan-core/pull/4360/commits/e3f97f0204573beba2a61ad11d50172012c9e017). but if we use typesript, we should always include such checks

Reply:
Maybe. For me it feels wrong, but maybe I just don't see the benefits.